backend/src/api/api.ts

```diff
@@ -75,6 +77,12 @@ const initApi: FastifyPluginAsync = async (fastify) => {
     },
     prefix: ApiPath.CATEGORIES,
   });
+  fastify.register(initCourseModulesApi, {
+    services: {
+      courseModule,
+    },
+    prefix: ApiPath.COURSES_$ID_MODULES,
```

Comment:
```suggestion
    prefix: ApiPath.COURSES_MODULES,
```
it should be a root of API entity